packages/netml/netml.0.1.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version : "1.2"
+name         : "netml"
+version      : "0.1.0"
+maintainer   : "Xavier Guérin <github@applepine.org>"
+authors      : "Xavier Guérin <github@applepine.org>"
+homepage     : "https://github.com/xguerin/netml"
+dev-repo     : "https://github.com/xguerin/netml.git"
+bug-reports  : "https://github.com/xguerin/netml/issues"
+license      : "ISC"
+
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+
+depends: [
+  "ocamlbuild"          {build}
+  "oasis"               {build & >= "0.4"}
+  "ocamlfind"           {build & >= "1.3.2"}
+  "js-build-tools"      {build}
+  "bitstring"           {build & >= "2.1.0"}
```

Comment:
I believe this should be a link-time dependency and not build time also, since it is the runtime support for ppx_bitstring.